README.md

```diff
@@ -385,12 +390,11 @@ documents = list(mongo_db[mongo_loader.metadata_collection].find(query))
 More on how to query collections can be found
 <a href="https://www.w3schools.com/python/python_mongodb_query.asp">here</a>.
 
-
 If the metadata associated with an already uploaded basket has not been uploaded to mongo, basket uuids can be passed to mongo_loader.load_mongo as shown below to upload missing basket metadata.
 
 ```python
-# uuids is a list of uuids (str) to add their data to the mongo db
-mongo_loader.load_mongo(uuids) 
+# uuids (str) is a list of uuids that will add the data of their associated baskets to the mongo db
```

Comment:
UUIDs of baskets whose metadata has not yet been loaded into MongoDB.

instead of uuids (str) is a list of uuids that will add the data of their associated baskets to the mongo db